src/table/virtualizedDataTable.jsx

```diff
@@ -22,42 +26,375 @@ const DEFAULT_HIGHLIGHT_ROW_COLOR = '#B3E5FC';
 
 export const IGNORE_EVENT = 'ignore-event';
 
+const KEYS_TO_SNIFF = [
+  'ArrowLeft',
+  'ArrowRight',
+  'ArrowUp',
+  'ArrowDown',
+  'Backquote',
+  'Backslash',
+  'Backspace',
+  'BracketLeft',
+  'BracketRight',
+  'Comma',
+  'Delete',
+  'Enter',
+  'Equal',
+  'Escape',
+  'Minus',
+  'Period',
+  'Quote',
+  'Semicolon',
+  'Slash',
+  'Space',
+  'Tab',
+];
+for (let keyOn = 0; keyOn < 10; keyOn += 1) {
+  KEYS_TO_SNIFF.push(`Digit${keyOn}`);
+}
+for (let keyOn = 48; keyOn < 91; keyOn += 1) {
+  KEYS_TO_SNIFF.push(`Key${String.fromCharCode(keyOn)}`);
+}
+
```

Comment:
Next time I need to make changes here I will switch to styled components and add support for them as well